src/book/03-guides/04-general/01-learning-resources.mdx

```diff
@@ -0,0 +1,180 @@
+---
+section: Guides
+chapter: General
+title: Learning Resources
+description: Where to find out how to learn about the various aspects of NUbots.
+slug: /guides/general/learning-resources
+---
+
+This page aims to provide direction and resources on where to go to learn more about various areas of NUbots. There is a mixture of resources available, including external resources and material prepared by NUbots team members. The main resources are Reading pages, videos and workshops. Workshops are run in lab, or on Zoom when not possible to do in the lab, by a member of the team. LinkedIn Learning has many great videos, some of which are linked here - as a UoN student, you can get access to LinkedIn Learning for free.
```

Comment:
```suggestion
This page aims to provide direction and resources on where to go to learn more about various areas of NUbots. There is a mixture of resources available, including external resources and material prepared by NUbots team members. The main resources are reading pages, videos and workshops. Workshops are run in lab, or on Zoom when not possible to do in the lab, by a member of the team. LinkedIn Learning has many great videos, some of which are linked here - as a UoN student, you can get access to LinkedIn Learning for free.
```

Reply:
lol this is actually meant to be 'NUbook pages', the 'Reading' heading used to be 'NUbook' and I just did a find and replace 😆